### PR TITLE
ES Lint rules

### PR DESCRIPTION
**Pull request by calebgregory**
[What rules are out there?](http://eslint.org/docs/rules/)

### CODE REVIEW

**Comment by calebgregory** [comment]
@sscotth I made a couple changes
, and I await your approval.

File: .eslintrc [hunk 1]
```diff
@@ -9,8 +9,9 @@ extends: eslint:recommended
 root: true
 
 rules:
+  comma-style: [2, "last"]
```
Comment (sscotth) [inline .eslintrc]: quotes are not needed for yaml strings

File: .eslintrc [hunk 2]
```diff
@@ -9,8 +9,9 @@ extends: eslint:recommended
...
   eol-last: 2
-  indent: [2, 2]
+  indent: [2, 2, {"VariableDeclarator": 2}]
```
Comment (sscotth) [inline .eslintrc]: I'd prefer a rule to force one var declaration per line. This style follows what the node app currently use and is how the predominant node style guide suggests:
http://nodeguide.com/style.html#variable-declarations

**Comment by sscotth** [comment]
Greg and Chad also are breaking a few of my expected style rules here: https://github.com/ccates23/minitwit/blob/delete/app/delete_feature/delete.test.js

Maybe we can add some additional rules to watch

**Comment by calebgregory** [comment]
@sscotth 'Ello gov'na!

**Comment by calebgregory** [comment]
@sscotth I'm looking for a rule for formatting callback spacing.  Haven't found it yet, but I'm [thinking about this...](http://eslint.org/docs/rules/prefer-arrow-callback)
Thoughts?

**Comment by sscotth** [comment]
Just noticed this comment. I tend to not look at closed PRs. Can you give some examples on what you are looking for?

**Pull request merged**